lib/runner/reporters/junit.js

```diff
@@ -56,6 +56,22 @@ module.exports = new (function() {
 
     adaptAssertions(module);
 
+    var filename = moduleName + '.xml';
```

Comment:
Okay, I found a reasonable template of what the output should look like.  I'm going to redo this with tests that verify the current output and then test the change.